Command/winzouStateMachineDebugCommand.php

```diff
@@ -123,4 +124,32 @@ protected function printTransitions(array $transitions, OutputInterface $output)
 
         $table->render();
     }
+
+    /**
+     * @param array           $allCallbacks
+     * @param OutputInterface $output
+     */
+    protected function printCallbacks(array $allCallbacks, OutputInterface $output)
+    {
+        foreach ($allCallbacks as $type => $callbacks) {
+            if ($type === 'before' || $type === 'after') {
```

Comment:
Why do you filter by type? Can you just display all types of callbacks?

Reply:
Well I didn't notice that guard callbacks were here because I'm using Sylius as example. Sylius doesn't use guard callbacks. I'll remove this useless check